test/hotspot/jtreg/vmTestbase/nsk/jdi/ThreadReference/interrupt/interrupt001a.java

```diff
@@ -205,12 +206,14 @@ private static int checkInterruptStatus() {
     }
 }
 
-class interrupt001aThread extends Thread {
+class interrupt001aThread implements Runnable {
```

Comment:
Does it make sense to rename class to something like Interrupt001aTask? Just to let know that it is not a thread. Also, there is ./nsk/share/jdi/NamedTask.java for task with names in JDI tests.

Reply:
Ok. I've made both these changes.